From: =?utf-8?q?G=C3=B6ran_Weinholt?= <weinholt@debian.org>
Date: Sun, 23 Sep 2018 23:10:31 +0200
Subject: Clean up extra files in the clean_all target

This cleans up a few additional autogenerated files in the clean_all target.
---
 Makefile.skel | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.skel b/Makefile.skel
index 957f0f5..cbae75f 100644
--- a/Makefile.skel
+++ b/Makefile.skel
@@ -66,4 +66,6 @@ clean_all: clean
 	rm -f src/promemul/Makefile src/include/Makefile
 	rm -f src/useremul/Makefile src/include/Makefile
 	rm -f src/console/Makefile src/symbol/Makefile
-
+	rm -f src/machines/automachine.cc
+	rm -f doc/machines.html doc/components.html
+	rm -f $(patsubst %.SKEL,%,$(wildcard doc/machines/*.SKEL))
